.env.sample

```diff
@@ -1,14 +1,4 @@
 # edit and save as .env
-COMPOSE_FILE=docker-compose.base.yml:docker-compose.dev.yml
 TZ=Europe/Berlin
-DOMAINNAME=docker.localhost
```

Comment:
why? dev configuration might be simpler with this environment variable

Reply:
It's only used twice in the dev environment. 
Prod only needs the domainname in the static yml, which can't be templated. 
Plus in dev it should be "Docker.localhost" in most cases.